examples/undo-tree/index.jsx

```diff
@@ -0,0 +1,53 @@
+import Microcosm from 'Microcosm'
+import React from 'react'
+import Drawing from './components/drawing'
+
+class UndoTree extends Microcosm {
+
+  shouldHistoryKeep(transaction) {
+    return true
+  }
```

Comment:
you could always go the verbose route `shouldMergeTransactionIntoBaseStateAndPurgeFromHistory`